dist/linux/ghostty_nautilus.py

```diff
@@ -0,0 +1,97 @@
+# Adapted from wezterm: https://github.com/wez/wezterm/blob/main/assets/wezterm-nautilus.py
+# original copyright notice:
+#
+# Copyright (C) 2022 Sebastian Wiesner <sebastian@swsnr.de>
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+from os.path import isdir
+from gi import require_version
+from gi.repository import Nautilus, GObject, Gio, GLib
+
+
+class OpenInGhosttyAction(GObject.GObject, Nautilus.MenuProvider):
+    def __init__(self):
+        super().__init__()
+        session = Gio.bus_get_sync(Gio.BusType.SESSION, None)
+        self._systemd = None
+        # Check if the this system runs under systemd, per sd_booted(3)
+        if isdir('/run/systemd/system/'):
+            self._systemd = Gio.DBusProxy.new_sync(session,
+                    Gio.DBusProxyFlags.NONE,
+                    None,
+                    "org.freedesktop.systemd1",
+                    "/org/freedesktop/systemd1",
+                    "org.freedesktop.systemd1.Manager", None)
+
+    def _open_terminal(self, path):
+        cmd = ['ghostty', f'--working-directory={path}', '--gtk-single-instance=false']
```

Comment:
Probably should set a unique `--class` and `--x11-instance-name` so that DBus doesn't get confused with multiple instances of Ghostty running.